README.md

```diff
@@ -84,7 +84,7 @@ https://cn.dapr.io/meeting/
 
 Every week, we meet to plan and sync on the progress for the milestone's release. This meeting includes maintainers and active contributors and usually lasts for about 30-60 minutes. If you would like to **actively contribute** towards a milestone, ask a maintainer for the meeting's password.
 
-**Zoom Call:** https://us02web.zoom.us/j/81902234964<br>
+**Zoom Call:**https://zoom.us/j/91940016938?pwd=bGNRVmlPK094a0tQZWRlTTJIZUl6UT09<br>
```

Comment:
Please, don't add the password here. See in the comment that this  password is given on demand only.